pkg/cmd/types_test.go

```diff
@@ -9,25 +9,53 @@ func TestValidate(t *testing.T) {
 	noURI.URI = ""
 
 	dockerCfgAndUsername := NewDefaultImageInspectorOptions()
+	dockerCfgAndUsername.Image = "image"
```

Comment:
:+1: I had it on my list to change this.  Thanks!  (it was preventing 100% coverage)